package.nls.cs.json

```diff
@@ -1,33 +1,35 @@
 {
-	"extension.description": "Debug your JavaScript code in the Edge browser, or any other target that supports the Edge Debugger protocol.",
```

Comment:
Where did these translations come from?

If we have missing translations, I can reach out to our localization lead. I'm not sure these nls files are actually used in the final packaged build

Reply:
Removed the autogen loc chagnes.